app/views/application/_flash_content.html.erb

```diff
@@ -1,29 +1,34 @@
-<% flash.each do |name, msg| %>
-  <% if msg.include?("unable") %>
+<% if alert %>
```

Comment:
Does this work? I would have expected you to need `flash.alert` or `flash.notice`. Nice if these are existing helper methods though!